### PR TITLE
Fix display issues with search bar

### LINKED ISSUE #38807: Navigation "hamburger" broken on mobile layout
E.g., https://docs.openshift.com/container-platform/4.9/welcome/index.html

The hamburger menu appears to have it's `z-index` too low, and is therefore hidden under the header

image

Then scrolling down, it finally comes out of hiding

image

Before finally becoming usable

image

This was on a Chrome browser, on a desktop machine with the window-width fully minimized. Same thing happened on a Chrome browser on an iPhone.

There's also an issue I noticed using the "Google Search" app on my iPhone, where the hamburger menu isn't actually clickable, and doesn't scroll with page, just stays at the top. I can't replicate outside of the specific "Google Search" app.

### LINKED ISSUE #40909: Weird CSS issue - No search bar visible on Safari
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.9/welcome/index.html

### What needs fixing?
Search bar not visible in Safari - unable to search through documentation for key words as a result - only visible when on Chrome.

**VERSION:**
Safari v14.1.2 (16611.3.10.1.6)

**IMAGES:**
https://imgur.com/a/7A0awOc

### PR DESCRIPTION
**Pull request by wgordon17**
Closes #38807

### CODE REVIEW

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -270,21 +286,29 @@
   <!-- adjust page css based on breadcrumb and/or resize -->
   <script type="text/javascript">
     window.addEventListener('DOMContentLoaded', () => {
-      if (window.innerWidth >= 1008) {
+      if ($(window).width() >= 1008) {
```
Comment (wgordon17) [inline _templates/_page_openshift.html.erb:289]: The switch here away from `innerWidth` addresses a Safari issue where `innerWidth` isn't actually accurate. So there was an unknown bug in Safari where resizing the browser caused an issue where the sidebar was shown and hidden at the same time

Comment (wgordon17) [reply _templates/_page_openshift.html.erb:289]: https://user-images.githubusercontent.com/22222756/154157516-941784d2-31de-47f0-9dfd-06d6955ac9af.mov

File: _templates/_page_openshift.html.erb [hunk 2]
```diff
@@ -48,6 +48,22 @@
     }
   </script>
 
+  <style>
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:51]: We should move this into docs.css.

**Comment by aireilly** [comment]
A preview with an updated `docs.css` is here: http://file.emea.redhat.com/aireilly/files/fix-search-bar/welcome/index.html

**Pull request merged**